packages/vscode-ext/client/src/client.ts

```diff
@@ -13,7 +13,7 @@ let client: LanguageClient;
 export const startClient = (context: vscode.ExtensionContext) => {
   // The server is implemented in node
   let serverModule = context.asAbsolutePath(
-    path.join("server", "out", "server.js")
+    path.join("dist", "server", "src", "server.js")
```

Comment:
Have you checked that server still starts?

I think it's still used for minor stuff (semantic highlighting), so it's possible that the extension is mostly working but this part doesn't.

Reply:
> Have you checked that server still starts?
> 
> I think it's still used for minor stuff (semantic highlighting), so it's possible that the extension is mostly working but this part doesn't.

![image](https://user-images.githubusercontent.com/6453661/194707411-f7f9b8e4-2333-4e3d-92e4-6aba82f938d3.png)

Looks like it does.

Reply:
The _semantic_ highlighting is different from the basic textmate grammar highlighting (only semantic highlighting requires the language server).

I took a look and semantic highlighting is broken, but for the unrelated reasons, it doesn't work on 0.5 either. The server works, or at least I see the `.../dist/server/src/server.js` in the process list, so it's fine.

Reply:
Okay, I've assumed that server provided info on difference between `normal` and `asdf`, otherwise syntactically similar — thus assuming it works

Reply:
Oh wait, I'm mistaken, it's not broken. I'm also mistaken about semantic highlighting, that's not done through the language server either yet. I have no idea why `normal` and `asdf` are highlighted differently on your machine, on mine it looks like this:

<img width="183" alt="Screen Shot 2022-10-08 at 16 56 58" src="https://user-images.githubusercontent.com/89368/194708689-ccb13925-b286-4bc4-919e-f0621e01c721.png">

The easiest way to check is to add a syntax error, those are reported through the language server only. That's the only useful thing the language server does for now.

Reply:
![image](https://user-images.githubusercontent.com/6453661/194709567-5914a8d8-b666-4528-9abb-452dcb94eb98.png)
Yes, I do get those in this build)

Reply:
(Alas that looks more like a parser error)